src/client/components/PasswordForm.tsx

```diff
@@ -83,20 +100,33 @@ const passwordInput = css`
 
 const TooLong = () => {
   return (
-    <div>
-      <div css={[baseIconStyles, failureIconStyles]}>
-        <SvgCross />
+    <div css={feedbackMessageStyles}>
+      <div css={[feedbackMessageIconStyles, errorIconStyles]}>
+        <SvgAlertTriangle />
+      </div>
+      <div css={[baseMessageStyles, redText]}>
+        {ShortPasswordFieldErrors.MAXIMUM_72}
       </div>
-      <div css={baseMessageStyles}>{ShortPasswordFieldErrors.MAXIMUM_72}</div>
     </div>
   );
 };
 
 const TooShort = () => {
+  const infoIconStyles = css`
+    svg {
+      fill: ${neutral[46]};
+      border-radius: 50%;
+      // Bit of a hack - the SVG for the tick icon is positioned more fully
+      // inside its viewBox than the SVG for the info icon, so this forces the
+      // info icon to take up the same amount of pixel space.
```

Comment:
css comments use `/* ... */` syntax

```suggestion
      /* Bit of a hack - the SVG for the tick icon is positioned more fully
       inside its viewBox than the SVG for the info icon, so this forces the
       info icon to take up the same amount of pixel space. */
```

Reply:
https://css-tricks.com/snippets/css/comments-in-css/